query-node/build.sh

```diff
@@ -11,19 +11,23 @@ set +a
 
 yarn clean
 
-# We generate the code for each service separately to be able to specify
-# separate database names.
+# Install hydra codegen in separate dir to avoid dependency clashes
+cd ./codegen
+yarn
+cd ..
 
-# Build indexer customizing DB name
-DB_NAME=${INDEXER_DB_NAME} yarn codegen:indexer
+# Generate types and server code
+TYPEGEN_WS_URI="${TYPEGEN_WS_URI:-ws://localhost:9944}" yarn typegen:configure
+yarn typegen
+yarn codegen:noinstall
+yarn format
 
-# Build graphql-server customizing DB name
-DB_NAME=${PROCESSOR_DB_NAME} yarn codegen:server
+# FIXME: Tmp add "declaration": true to query-node tsconfig
+sed -i 's/"compilerOptions": {/"compilerOptions": {\n    "declaration": true,/' ./generated/graphql-server/tsconfig.json
```

Comment:
```suggestion
sed -i.bak 's/"compilerOptions": {/"compilerOptions": {\n    "declaration": true,/' ./generated/graphql-server/tsconfig.json
rm ./generated/graphql-server/tsconfig.json.bak
```

Unfortunately, the \`sed -i \` does not work on Mac :disappointed: The [best we can do](https://stackoverflow.com/a/22084103/2273046) here is to use `.bak` file and remove it (optionally).

Reply:
This shouldn't be necessary anymore with the 2.0.1-beta.17 version.